content/manuals/engine/install/fedora.md

```diff
@@ -116,8 +116,8 @@ $ sudo dnf-3 config-manager --add-repo {{% param "download-url-base" %}}/docker-
    ```console
    $ dnf list docker-ce --showduplicates | sort -r
 
+   docker-ce.x86_64    3:27.4.0-1.fc41    docker-ce-stable
```

Comment:
:heart: These are always the cumbersome ones. Not  sure if we can find an easier way to handle that though. I think it's the good thing to do (at some point we were showing examples with multiple versions behind, which also isn't great).